### DISCUSSION TITLE
Screenshot How

### DISCUSSION THREAD

**Discussion by pigasspig** [leftwm/leftwm#459] [Q&A]
I would like to know how to take a screenshot

**Comment by VuiMuich** [comment]
I use `scrot` for screen shots, and `obs` or `ffmpeg` to record videos.

**Reply by pigasspig** [reply]
how do u install scrot?

**Reply by VuiMuich** [reply]
You may want to read this:

https://wiki.archlinux.org/title/Pacman#Installing_packages

And maybe also this:

https://wiki.archlinux.org/title/Screen_capture#scrot

You will notice scrot is part of the 'community' repo of arch Linux. 
If it wasn't it most probably would be part of the [AUR](https://aur.archlinux.org/). I prefer `paru` as my [AUR helper](https://wiki.archlinux.org/title/AUR_helpers).
